src/engine/strat_engine/backstore/util.rs

```diff
@@ -55,11 +60,12 @@ fn get_all_empty_devices() -> StratisResult<Vec<PathBuf>> {
     Ok(enumerator
         .scan_devices()?
         .filter(|dev| {
-            !((dev.property_value("ID_PART_TABLE_TYPE").is_some()
-                && dev.property_value("ID_PART_ENTRY_DISK").is_none())
-                || dev.property_value("ID_FS_USAGE").is_some())
+            dev.property_value("DM_MULTIPATH_DEVICE_PATH").is_none()
+                && !((dev.property_value("ID_PART_TABLE_TYPE").is_some()
+                    && dev.property_value("ID_PART_ENTRY_DISK").is_none())
+                    || dev.property_value("ID_FS_USAGE").is_some())
         })
-        .map(|i| i.devnode().expect("block devices have devnode").into())
+        .filter_map(|i| i.devnode().map(|d| d.into()))
```

Comment:
AFAICT, filtering on the devnode has nothing to do w/ multipath. Please remove it from this commit. It occurs at this line and also line 82.